### DISCUSSION TITLE
Happymodel ELRS F4 2G4 AIO 5in1 Issues

### DISCUSSION THREAD

**Discussion by Neukom720** [ExpressLRS/ExpressLRS#741] [Q&A] [answered]
Hey guys was wondering if anyone knew much about this fc+receiver the information from happymodel is terrible and everything is firmware targets that were released early and with absolutely no manual. Now I finally got it to bind by switching it to 500hz in the lua script but now after I got it bound I cant change it back to 250hz because it drops all connection and no matter what I do it will not connect at 250hz and because I have a Tango 2, 500hz does not work for me because it obviously just drops packets and sits at around 215hz when connected at 500hz as it reports at the top of the lua script. If the tango 2 could do 500hz I wouldn't care but of course it cant because tbs said freedomtx was temporary fork of opentx.....yea right its never gonna fork back over! Also, not sure what elrs firmware to put onto the tx module im using so that it matches the rx but it is bound and connected on the current 1.0.0 release on the tx now. Please any help or insight would be greatly appreciated!

**Comment by Neukom720** [comment, marked as answer]
found the answer! for anyone else that needs 250hz this is what you enter in the betaflight cli

set expresslrs_rate_index = 1
save

here is where I found the info I needed

https://github.com/ExpressLRS/ExpressLRS-Configurator/issues/31

**Comment by JyeSmith** [comment]
https://github.com/betaflight/betaflight/pull/10788
